h/streamer/websocket.py

```diff
@@ -71,7 +71,7 @@ def handle_message(message):
             jsonschema.validate(payload, filter.SCHEMA)
 
             # Add backend expands for clauses
-            _expand_clauses(payload)
+            _expand_clauses(socket.request, payload)
```

Comment:
Now that I can actually run websockets. It is the same DB session.
Considering the short discussion on Slack regarding passing through the session all the way, which would unfortunately mean that the `h.api.storage` functions would have to accept the request and the session in addition to other arguments. Is it still worth it?

The other solution I could come up with is that the `h.api.storage` function could accept an optional DB session (as a keyword argument) for exactly these use cases where it's hard to follow.

Thoughts?